scraper.js

```diff
@@ -2,154 +2,19 @@ const dotenv = require("dotenv");
 //note: this only works locally; in Lambda we use environment variables set manually
 dotenv.config();
 
-const chromium = require("chrome-aws-lambda");
-const { addExtra } = require("puppeteer-extra");
-const Puppeteer = addExtra(chromium.puppeteer);
+const scrapers = require("./site-scrapers");
+const scraperCommon = require("./scraper_common.js");
 
-const { getAllCoordinates } = require("./getGeocode");
```

Comment:
I didn't check that every line of this code got moved over, but all the major pieces look right.

Reply:
**Methodology:**  I started with the prior `scraper.js` that had all of the Puppeteer calls in it. Then I compared that with prior `scrapers-no-browser.js` to see what got removed.  Then I added `if (usePuppeteer)` checks around the missing code.